src/SourceCode.Clay.OpenApi/Contact.cs

```diff
@@ -5,32 +5,72 @@
 
 #endregion
 
+using SourceCode.Clay.OpenApi.Serialization;
 using System;
+using System.Json;
 using System.Net.Mail;
 
 namespace SourceCode.Clay.OpenApi
 {
     /// <summary>
     /// Contact information for the exposed API.
     /// </summary>
-    public class Contact : IEquatable<Contact>
+    public sealed class Contact : IEquatable<Contact>
     {
+        #region Fields
+
+        private readonly JsonObject _json;
+
+        #endregion
+
         #region Properties
 
         /// <summary>
         /// Gets the identifying name of the contact person/organization.
         /// </summary>
-        public string Name { get; }
+        public string Name
+        {
+            get
+            {
+                if (_json.TryGetValue(OpenApiSerializer.PropertyConstants.Name, out var jv))
+                    return jv;
+
+                return null;
+            }
+        }
 
         /// <summary>
         /// Gets the URL pointing to the contact information.
         /// </summary>
-        public Uri Url { get; }
+        public Uri Url
+        {
+            get
+            {
+                if (_json.TryGetValue(OpenApiSerializer.PropertyConstants.Url, out var jv))
+                    return new Uri(jv);
+
+                return null;
+            }
+        }
 
         /// <summary>
         /// Gets email address of the contact person/organization.
         /// </summary>
-        public MailAddress Email { get; }
+        public MailAddress Email
+        {
+            get
+            {
+                if (_json.TryGetValue(OpenApiSerializer.PropertyConstants.Email, out var jv))
+                    return new MailAddress(jv);
+
+                return null;
+            }
+        }
+
+        /// <summary>
+        /// Gets custom properties.
+        /// </summary>
+        public VendorExtensions VendorExtensions { get; }
```

Comment:
This example uses the `VendorExtensions : IReadOnlyDictionary` pattern to avoid mutation